grails-rundeck-data-shared/src/main/groovy/rundeck/data/util/JobTakeoverQueryBuilder.groovy

```diff
@@ -42,4 +45,10 @@ class JobTakeoverQueryBuilder {
         return qry
     }
 
+    static boolean isValidUUID(String val) {
+        if (null == val) return false;
+        try { return null!= UUID.fromString(val) } catch (IllegalArgumentException e) {
+            return false
+        }
+    }
```

Comment:
fyi, `org.rundeck.utils.UUIDs.isValid(String)` does a more extensive validation. `1-1-1-1-1` is a valid UUID, but we expect them to be 36 characters in length